server.js

```diff
@@ -21,7 +19,44 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const endpoints = expressListEndpoints(app);
+  res.json(endpoints);
+});
```

Comment:
when you list the endpoints you can ass more info to document your API, for example you can filter the data by queries like `region` or even `lowestprice` (check the naming convention), you could add something like this:

```javascript
app.get("/", (req, res) => {
  const usageInfo = {
    description: "Get avocado sales data.",
    queryParameters: {
      region: "Filter by region (case-insensitive).",
      lowestprice:
        "Filter out all entries at a price point higher than the specified value.",
    },
  }
  const endpoints = expressListEndpoints(app);
  const avocadoEndpointIndex = endpoints.findIndex(
    (endpoint) => endpoint.path === "/avocado"
  )

  if (avocadoEndpointIndex !== -1) {
    endpoints[avocadoEndpointIndex].usageInfo = usageInfo
  }

  res.json(endpoints)
})
``` 

Have you tested this endpoint with this query param? JavaScript is case-sensitive, so `req.query.lowestprice` will look for a query parameter named "lowestprice" exactly as it's written in the URL. If you're sending the request with a query parameter named "lowestPrice" (with a capital "P"), then `req.query.lowestprice` won't be able to retrieve its value.
You can fix your code using the correct case for the query parameter:

```javascript
const priceSearch = req.query.lowestPrice;
```